src/web/app/pages-admin/admin-notifications-page/admin-notifications-page.component.ts

```diff
@@ -199,6 +182,26 @@ export class AdminNotificationsPageComponent implements OnInit {
     );
   }
 
+  /**
+   * Deletes a notification.
+   */
+  deleteNotificationHandler(notificationId: string): void {
+    this.notificationService.deleteNotification(notificationId)
+      .subscribe(
+        () => {
+          this.statusMessageService.showSuccessToast('Notification deleted successfully.');
+          this.notificationsTableRowModels = this.notificationsTableRowModels.filter(
```

Comment:
```suggestion
        () => {
          this.statusMessageService.showSuccessToast(resp.message);
          this.notificationsTableRowModels = this.notificationsTableRowModels.filter(
```
Not wrong, but can simply use the message from the response.

Reply:
Thank you very much! I have updated that in the latest commit.